docker-compose.yml

```
@@ -0,0 +1,98 @@
+version: '2.4'
+
+services:
+  myskills-api:
+    image: jobtechswe/myskills-api:latest
+    environment:
+      - REDIS_API_HOST=myskills-redis
+      - REDIS_API_PORT=6379
+      - SERVER_PORT=3000
+      - DOMAIN_HOST=myskills-api
+      - MYDATA_OPERATOR=http://mydata-operator:4000
+      - MYDATA_APP=http://mydata-app:1337
+    depends_on:
+      - mydata-operator
+      - mydata-app
+      - myskills-redis
+    ports:
+      - 127.0.0.1:3000:3000
+    networks:
+      myskills:
+        aliases:
+          - myskills-api
+  myskills-redis:
+    image: redis:5.0
+    ports:
+      - 127.0.0.1:6380:6379
+    networks:
+      myskills:
+        aliases:
+          - myskills-redis
+  mydata-operator-redis:
+    image: redis:5.0
+    ports:
+      - 127.0.0.1:6379:6379
+    networks:
+      myskills:
+        aliases:
+          - mydata-operator-redis
+  mydata-operator-postgres:
+    image: postgres:11.2
+    ports:
+      - 127.0.0.1:5432:5432
+    networks:
+      myskills:
+        aliases:
+          - mydata-operator-postgres
+    environment:
+      - POSTGRES_USER=postgresuser
+      - POSTGRES_PASSWORD=postgrespassword
+      - POSTGRES_DB=mydata
+      - POSTGRES_INITDB_ARGS=--encoding=utf8
+    healthcheck:
+      test: ['CMD-SHELL', 'pg_isready -U postgresuser -d mydata']
+      interval: 10s
+      timeout: 5s
+      retries: 5
+  mydata-operator:
+    image: jobtechswe/mydata-operator:latest-tag
```

This could probably be `0.25.2` to match up with API!

```suggestion
    image: jobtechswe/mydata-operator:0.25.2
```